run_dir/design/qpcr_pools.html

```diff
@@ -49,7 +49,7 @@ <h1><span id="page_title">qPCR queues</span>
 </div>
 
 <script src="/static/js/clipboard.min.js"></script>
```

Comment:
I think we could remove this one if we use the datatables extension instead? That would need some testing as well though

Reply:
Yes, but I haven't implemented the new button for the qPCR page yet, should I?